content/english/python-pixel/Activities/activity-9.md

```diff
@@ -12,50 +12,88 @@ draft: false
 from PIL import Image
-#Open the cat image
+
+# Open the cat image
 img = Image.open("cat.png")
-width=img.size[0]
-height=img.size[1]
-#Set up a new image with the half width and height
-newimg=Image.new('RGB',(width,height) )
-#Set the pixel for the new image
-for i in range(width):         # For every col
-    for j in range(height):    # For every row
-            # Focus on how we get our heighNew.
-            heightNew = height - 1 - j
-            # Focus on how we get our widthNew.
-            widthNew = width - 1 - i
-            color = img.getpixel( (widthNew, heightNew) ) #get the color from original image
-            newimg.putpixel((i,j),color) # put the color into new image
+width = img.size[0]
+height = img.size[1]
+
+# Set up a new image with the half width and height
+newimg = Image.new('RGB',(width,height) )
+
+# Set the pixel for the new image
+for i in range(width): # For every column
+    for j in range(height): # For every row
+        # Focus on how we get our heightNew.
+        heightNew = height - 1 - j
+        
+        # Focus on how we get our widthNew.
+        widthNew = width - 1 - i
+
+        color = img.getpixel((widthNew, heightNew)) # Get the color from original image
+        newimg.putpixel((i,j),color) # Put the color into new image
+
 newimg.save("Mycat.png")
 ```
+
 Wow! This is our new cat after rotating.
 <img src="../../media/catrot.png" width=50%>
-<hr/>
 
 {{% notice note %}}
 
-Pay attention to how we get our heightNew and widthNew. Think about them carefully and try to visualize them.
-<br/>
+Pay attention to how we get our heightNew and widthNew. Think about those variables carefully and try to visualize them.
+
 For example, try to apply them on the following 4x4 letter group:
 
 <img src="../../media/table.png" width=15%>
 
-Then rotate it 180 degree clock-wise and compare it with the previous output. Are they the same?
+Then rotate it 180 degree clockwise and compare it with the previous output. Are they the same?
 {{% /notice %}}
-<hr/>
 
-### Challenge - Rotate your cat 90 degree counter-clock-wise
-<hr/>
 
-It's now your turn to rotate your cat 90 degree counter-clock-wise! Exciting！
+### Challenge - Rotate your cat 90 degrees counter-clock-wise
+
+It's now your turn to rotate your cat 90 degrees counter-clockwise! 
+
 <iframe height="600px" width="100%" src="https://repl.it/@nuevofoundation/Python-Pixel-Activity9?lite=true" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
-<br/><br/>
+
+{{% showanswer Advanced %}}
+
+The Pillow module has functions here that can help simplify the steps above! To do that, let’s look at the <b>Pillow Image</b> function <b>rotate()</b>.
+
+The rotate image works using angles. For example, <b>rotate(45)</b> will tilt your picture sideways by 45 degrees. Using <b>rotate(90)</b> will turn your picture sideways.
```

Comment:
Should all code be in `monospace` instead of **bold**

```suggestion
The rotate image works using angles. For example, `rotate(45)`will tilt your picture sideways by 45 degrees. Using `rotate(90)` will turn your picture sideways.
```